Strings/Strings.java

```diff
@@ -0,0 +1,199 @@
+    {
+        int length=0;
+        for(char c  :givenString)
+        {
+         length++;
+        }
+        return length;
+    }
+    /**
+     * Used to compare two strings
+     * @param firstString denotes the first string in the form of a character array which is to be compared
+     * @param secondString denotes the second string in the form of a character array which is to be compared
+     * @return 1 if the two strings are equal else returns 0
+     */
+    int stringCompare(char[] firstString,char[] secondString)
+    {
+        int flag=0;
+        if(stringLength(firstString)==stringLength(secondString))
+        {
+            for(int i=0;i<stringLength(firstString);i++)
+            {
+                if(firstString[i]==secondString[i])
+                    flag=1;
+                else
+                {
+                    flag=0;
+                    break;
+                }
+            }
+        }
+        else
+            flag= 0;
+        return flag;
+    }
+    /**
+     * reverseString method is used to reverse the entered string
+     * @param firstString denotes the first string in the form of a character array which is to be reversed
+     * @return the reversed string
+     */
+    char[] reverseString(char[] firstString)
+    {
+        int lengthOfString=stringLength(firstString),j=lengthOfString-1;
+        char[] reversesString=new char[lengthOfString];
+        for(int i=0;i<lengthOfString;i++,j--)
+        {
+            reversesString[i]=firstString[j];
+        }
+        return reversesString;
+    }
+    /**
+     * Converts the characters in a string to opposite case(Uppercase alphabets to lowercase alphabets and vice versa)
+     * @param firstString denotes the first string in the form of a character array which is to be converted to opposite case
+     * @return the converted string
+     */
+    char[] convertsCharactersToOppositeCaseInString(char[] firstString)
+    {
+        int lengthOfString=stringLength(firstString);
+        char[] newReplacedString=new char[lengthOfString];
+        for(int i=0;i<lengthOfString;i++)
+        {
+            if(firstString[i]>=65 && firstString[i]<=90)
+                newReplacedString[i]=(char)(firstString[i]+32);
+            else if(firstString[i]>=97 && firstString[i]<=122)
+                newReplacedString[i]=(char)(firstString[i]-32);
+            else
+                newReplacedString[i]=(char)firstString[i];
+        }
+        return newReplacedString;
+    }
+    /**
+     * Finds the largest word among a string
+     * @param firstString denotes the string in the form of a character array from which the largest word is to be derived
+     * @return the largest word of a string and If two words are of same length then return last word.
+     */
+    char[] largestWord(char[] firstString)
+    {
+        int lengthOfWord=0,lengthOfMaximumWord=0,index=0,i=0,j=0;
+        for(i=0,j=0;j<stringLength(firstString);i++,j++)
+        {

```

Comment:
I think you don't need to increment j each time. What you can do is you can leave J at the starting of each word thus whenever ' ' comes then only update J.
What you say?